cpp/src/FileSystemTest.cc

```diff
@@ -0,0 +1,295 @@
+/**
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <fstream>
+#include <iostream>
+#include <ctime>
+#include <iomanip>
+#include <chrono>
+#include <functional>
+#include <thread>
+
+#include "FileSystem.h"
+
+using namespace alluxio;
+
+void CreateDirectoryTest(FileSystem* fileSystem, const char* dirPath) {
+  std::cout << "TEST - CREATE DIRECTORY: " << std::endl;
+  Status status = fileSystem->CreateDirectory(dirPath);
+  if (status.ok()) {
```

Comment:
Do we have the branch and cout in unit test? Shouldn't you assert the expected output?